index.js

```diff
@@ -4,7 +4,7 @@ import MochaAdapter from './lib/adapters/MochaAdapter.js'
 import TapReporter from './lib/reporters/TapReporter.js'
 import ConsoleReporter from './lib/reporters/ConsoleReporter.js'
 import {Assertion, Test, Suite} from './lib/Data.js'
-import {autoRegister} from './lib/helpers.js'
```

Comment:
You don't need this default export for that. Instead do `import * as helpers from '...'`.